### DISCUSSION TITLE
Welcome to any-advice-issue Discussions!

### DISCUSSION THREAD

**Discussion by viaweb3** [kcc-community/any-advice-issue#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AgentBlueGoTerra** [comment]
Hello, I am Jared with Go-Terra. We currently run a Terra Validator, and are looking forward to sharing lessons learned in that process with this community.
Litao is the principle engineer for our project.

**Reply by viaweb3** [reply]
cool and welcome
